css-layout-classes/style.css

```diff
@@ -0,0 +1,23 @@
+.container {
+  max-width: 800px;
+  margin-left: auto;
+  margin-right: auto;
+  align-content: center;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-one-third {
+  flex-basis: calc(100% / 3);
+}
+
+.column-one-fourth {
+  flex-basis: calc(100% / 4);
+}
+
+.column-one-half {
+  flex-basis: calc(100% / 2);
+}
```

Comment:
Will need to add a selector for `.column-two-thirds` after other comments are addressed